### DISCUSSION TITLE
Cannot find module '@vue-storefront/prestashop-api'

### DISCUSSION THREAD

**Discussion by AlvarezJean** [vuestorefront/prestashop#25] [Q&A]
Hi,
my build is failing with that error: Cannot find module '@vue-storefront/prestashop-api';
So type declarations are missing.
Has anyone been stuck at this point?
Thanks

**Comment by samberrry** [comment]
Hi,
have you installed dependencies correctly?
have not you had error with that?

possibly provide screenshots.
thanks
